embulk-output-sqlserver/build.gradle

```diff
@@ -26,3 +26,38 @@ embulkPlugin {
     category = "output"
     type = "sqlserver"
 }
+
+publishing {
+    publications {
+        maven(MavenPublication) {
+            pom {  // https://central.sonatype.org/pages/requirements.html
+                developers {
+                    developer {
+                        name = "Hitoshi Tanaka"
+                        email = "thitoshi@cac.co.jp"
+                    }
+                    developer {
+                        name = "Megumi Tomita"
+                        email = "tomykaira@gmail.com"
+                    }
+                    developer {
+                        name = "uu59"
+                        email = "k@uu59.org"
+                    }
```

Comment:
@uu59 Hi!

As written in the pull-request description, we're trying to nominate developers who have contributed a important portion in `embulk-output-jdbc/mysql/postgresql/redshift/sqlserver`.

We wonder if you're okay to nominate you in `pom.xml` of `embulk-output-sqlserver` on Maven Central, from your work in #121. What do you think? 

I picked up your name and email from the Git commit log and/or your public GitHub user page. Please let me know if you don't want, or if you want to be listed in another name/email.